wp-content/themes/OCTA/it-framework/plugins/importer/content/theme_options.txt
a:1:{i:0;O:8:"stdClass":2:{s:11:"option_name";s:13:"theme_options";s:12:"option_value";s:17183:"a:495:{s:6:"layout";s:0:"";s:10:"main_width";s:4:"1170";s:13:"sidebar_width";s:2:"25";s:15:"content_padding";s:7:"100|100";s:11:"body_smooth";s:1:"1";s:13:"bodyfontcolor";s:0:"";s:14:"bodylinkscolor";s:0:"";s:11:"bodybgcolor";s:0:"";s:11:"bodybgimage";s:0:"";s:18:"body_bg_img_repeat";s:0:"";s:16:"body_bg_position";s:3:"0 0";s:12:"body_bg_size";s:7:"contain";s:20:"body_bg_img_parallax";s:1:"1";s:11:"usepatterns";s:0:"";s:13:"patterns-imgs";s:0:"";s:10:"animsition";s:0:"";s:18:"data-animsition-in";s:7:"fade-in";s:19:"data-animsition-out";s:8:"fade-out";s:16:"data-duration-in";s:4:"1000";s:17:"data-duration-out";s:4:"1000";s:11:"page_loader";s:1:"1";s:17:"page_loader_style";s:7:"circles";s:11:"loaderimage";s:0:"";s:8:"skin_css";s:5:"light";s:10:"skin_color";s:7:"#dd2f42";s:12:"show_top_bar";s:1:"1";s:12:"top_left_mod";s:1:"2";s:14:"top_bar_left_0";s:0:"";s:15:"top_left_menu_0";s:0:"";s:19:"top_left_menu_css_0";s:0:"";s:20:"top_left_menu_hide_0";s:1:"0";s:20:"top_left_text_icon_0";s:0:"";s:15:"top_left_text_0";s:0:"";s:19:"top_left_text_css_0";s:0:"";s:20:"top_left_text_hide_0";s:1:"0";s:20:"top_left_link_icon_0";s:0:"";s:20:"top_left_link_text_0";s:0:"";s:19:"top_left_link_url_0";s:0:"";s:22:"top_left_link_target_0";s:0:"";s:19:"top_left_link_css_0";s:0:"";s:20:"top_left_link_hide_0";s:1:"0";s:22:"top_left_socials_txt_0";s:14:"Follow us on: ";s:22:"top_left_socials_css_0";s:0:"";s:23:"top_left_socials_hide_0";s:1:"0";s:21:"top_left_search_css_0";s:0:"";s:22:"top_left_search_hide_0";s:1:"0";s:20:"top_left_login_css_0";s:0:"";s:22:"top_left_woocart_css_0";s:0:"";s:23:"top_left_woocart_hide_0";s:1:"0";s:19:"top_left_wpml_css_0";s:0:"";s:20:"top_left_wpml_hide_0";s:1:"0";s:14:"top_bar_left_1";s:4:"text";s:15:"top_left_menu_1";s:0:"";s:19:"top_left_menu_css_1";s:0:"";s:20:"top_left_menu_hide_1";s:1:"0";s:20:"top_left_text_icon_1";s:16:"icmon-headphones";s:15:"top_left_text_1";s:17:"(012) 345-567-890";s:19:"top_left_text_css_1";s:0:"";s:20:"top_left_text_hide_1";s:1:"0";s:20:"top_left_link_icon_1";s:0:"";s:20:"top_left_link_text_1";s:0:"";s:19:"top_left_link_url_1";s:0:"";s:22:"top_left_link_target_1";s:0:"";s:19:"top_left_link_css_1";s:0:"";s:20:"top_left_link_hide_1";s:1:"0";s:22:"top_left_socials_txt_1";s:14:"Follow us on: ";s:22:"top_left_socials_css_1";s:0:"";s:23:"top_left_socials_hide_1";s:1:"0";s:21:"top_left_search_css_1";s:0:"";s:22:"top_left_search_hide_1";s:1:"0";s:20:"top_left_login_css_1";s:0:"";s:22:"top_left_woocart_css_1";s:0:"";s:23:"top_left_woocart_hide_1";s:1:"0";s:19:"top_left_wpml_css_1";s:0:"";s:20:"top_left_wpml_hide_1";s:1:"0";s:14:"top_bar_left_2";s:4:"text";s:15:"top_left_menu_2";s:0:"";s:19:"top_left_menu_css_2";s:0:"";s:20:"top_left_menu_hide_2";s:1:"0";s:20:"top_left_text_icon_2";s:16:"fa fa-envelope-o";s:15:"top_left_text_2";s:16:"email@domain.com";s:19:"top_left_text_css_2";s:0:"";s:20:"top_left_text_hide_2";s:1:"0";s:20:"top_left_link_icon_2";s:0:"";s:20:"top_left_link_text_2";s:0:"";s:19:"top_left_link_url_2";s:0:"";s:22:"top_left_link_target_2";s:0:"";s:19:"top_left_link_css_2";s:0:"";s:20:"top_left_link_hide_2";s:1:"0";s:22:"top_left_socials_txt_2";s:14:"Follow us on: ";s:22:"top_left_socials_css_2";s:0:"";s:23:"top_left_socials_hide_2";s:1:"0";s:21:"top_left_search_css_2";s:0:"";s:22:"top_left_search_hide_2";s:1:"0";s:20:"top_left_login_css_2";s:0:"";s:22:"top_left_woocart_css_2";s:0:"";s:23:"top_left_woocart_hide_2";s:1:"0";s:19:"top_left_wpml_css_2";s:0:"";s:20:"top_left_wpml_hide_2";s:1:"0";s:13:"top_right_mod";s:1:"1";s:15:"top_bar_right_0";s:0:"";s:16:"top_right_menu_0";s:0:"";s:20:"top_right_menu_css_0";s:0:"";s:21:"top_right_menu_hide_0";s:1:"0";s:21:"top_right_text_icon_0";s:0:"";s:16:"top_right_text_0";s:0:"";s:20:"top_right_text_css_0";s:0:"";s:21:"top_right_text_hide_0";s:1:"0";s:21:"top_right_link_icon_0";s:0:"";s:21:"top_right_link_text_0";s:0:"";s:20:"top_right_link_url_0";s:0:"";s:23:"top_right_link_target_0";s:0:"";s:20:"top_right_link_css_0";s:0:"";s:21:"top_right_link_hide_0";s:1:"0";s:23:"top_right_socials_txt_0";s:14:"Follow us on: ";s:23:"top_right_socials_css_0";s:0:"";s:24:"top_right_socials_hide_0";s:1:"0";s:22:"top_right_search_css_0";s:0:"";s:23:"top_right_search_hide_0";s:1:"0";s:21:"top_right_login_css_0";s:0:"";s:23:"top_right_woocart_css_0";s:0:"";s:24:"top_right_woocart_hide_0";s:1:"0";s:20:"top_right_wpml_css_0";s:0:"";s:21:"top_right_wpml_hide_0";s:1:"0";s:15:"top_bar_right_1";s:7:"socials";s:16:"top_right_menu_1";s:0:"";s:20:"top_right_menu_css_1";s:0:"";s:21:"top_right_menu_hide_1";s:1:"0";s:21:"top_right_text_icon_1";s:0:"";s:16:"top_right_text_1";s:0:"";s:20:"top_right_text_css_1";s:0:"";s:21:"top_right_text_hide_1";s:1:"0";s:21:"top_right_link_icon_1";s:0:"";s:21:"top_right_link_text_1";s:0:"";s:20:"top_right_link_url_1";s:0:"";s:23:"top_right_link_target_1";s:0:"";s:20:"top_right_link_css_1";s:0:"";s:21:"top_right_link_hide_1";s:1:"0";s:23:"top_right_socials_txt_1";s:0:"";s:23:"top_right_socials_css_1";s:0:"";s:24:"top_right_socials_hide_1";s:1:"0";s:22:"top_right_search_css_1";s:0:"";s:23:"top_right_search_hide_1";s:1:"0";s:21:"top_right_login_css_1";s:0:"";s:23:"top_right_woocart_css_1";s:0:"";s:24:"top_right_woocart_hide_1";s:1:"0";s:20:"top_right_wpml_css_1";s:0:"";s:21:"top_right_wpml_hide_1";s:1:"0";s:12:"topbar_color";s:5:"light";s:8:"barcolor";s:0:"";s:12:"bariconcolor";s:0:"";s:10:"barbgcolor";s:0:"";s:9:"bar_image";s:0:"";s:14:"bar_img_repeat";s:0:"";s:11:"bar_bg_size";s:5:"cover";s:8:"bar_full";s:1:"0";s:13:"topbar_border";s:1:"1";s:13:"header_layout";s:7:"classic";s:18:"header-minimal-set";s:6:"simple";s:11:"header-dark";s:0:"";s:11:"header-full";s:0:"";s:12:"header-fixed";s:0:"";s:15:"header-position";s:6:"bottom";s:7:"top-pos";s:2:"46";s:10:"bottom-pos";s:1:"0";s:14:"header_padding";s:3:"0|0";s:11:"show_search";s:1:"1";s:16:"search_box_style";s:0:"";s:9:"show_cart";s:1:"1";s:10:"header-ads";s:0:"";s:14:"header-ads-url";s:0:"";s:17:"show_head_socials";s:1:"1";s:14:"show_head_copy";s:1:"1";s:17:"nav_text_bg_color";s:0:"";s:14:"nav_icon_color";s:0:"";s:12:"nav_bg_color";s:0:"";s:9:"nav_image";s:0:"";s:18:"nav_img_full_width";s:0:"";s:14:"nav_img_repeat";s:0:"";s:16:"nav_border_color";s:0:"";s:16:"nav_border_width";s:0:"";s:16:"sticky_header_on";s:1:"1";s:15:"sticky_bg_color";s:0:"";s:17:"sticky_text_color";s:0:"";s:14:"sticky_tablets";s:1:"0";s:14:"sticky_mobiles";s:1:"0";s:13:"logo-position";s:0:"";s:9:"logo_type";s:5:"image";s:17:"header_logo_image";s:62:"http://wp.it-rays.net/octa/wp-content/uploads/2017/03/logo.png";s:11:"sticky_logo";s:0:"";s:9:"logo_font";s:7:"Raleway";s:14:"logo_font_size";s:4:"32px";s:16:"logo_font_weight";s:3:"900";s:15:"logo_font_color";s:0:"";s:16:"mobile_logo_size";s:4:"25px";s:11:"slogan_font";s:9:"Open Sans";s:16:"slogan_font_size";s:4:"11px";s:18:"slogan_font_weight";s:3:"400";s:17:"slogan_font_color";s:0:"";s:14:"nav_text_color";s:0:"";s:12:"header-icons";s:0:"";s:16:"header-sub-icons";s:1:"1";s:15:"header-subtitle";s:0:"";s:14:"sub_menu_color";s:0:"";s:17:"sub_menu_bg_color";s:0:"";s:19:"sub_menu_font_color";s:0:"";s:9:"locations";s:1:"6";s:10:"location_0";s:0:"";s:10:"location_1";s:9:"Mega Menu";s:10:"location_2";s:12:"Fashion Menu";s:10:"location_3";s:15:"Restaurant Menu";s:10:"location_4";s:12:"Wedding Menu";s:10:"location_5";s:12:"Medical Menu";s:10:"location_6";s:18:"404 Not Found Menu";s:17:"mob_menu_bg_color";s:7:"#dd3333";s:18:"mob_menu_txt_color";s:7:"#ffffff";s:12:"footer_style";s:1:"1";s:12:"footer_light";s:0:"";s:12:"footer_fixed";s:0:"";s:13:"foot_mid_show";s:1:"1";s:18:"foot_mid_widg_cols";s:1:"3";s:17:"foot_mid_bg_color";s:0:"";s:19:"foot_mid_top_border";s:0:"";s:19:"foot_mid_text_color";s:0:"";s:11:"foot_mid_bg";s:0:"";s:20:"foot_mid_bg_position";s:3:"0 0";s:18:"foot_mid_bg_repeat";s:0:"";s:16:"foot_mid_bg_size";s:5:"cover";s:16:"foot_bottom_show";s:1:"1";s:18:"foot_bot_widg_cols";s:2:"12";s:15:"foot_copyrights";s:77:"Copyrights © 2017. All rights reserved to <b class="main-color">OCTA</b> Co.";s:17:"foot_bot_bg_color";s:0:"";s:19:"foot_bot_text_color";s:0:"";s:11:"foot_bot_bg";s:0:"";s:18:"foot_bot_bg_repeat";s:0:"";s:20:"foot_bot_bg_position";s:7:"50% 50%";s:16:"foot_bot_bg_size";s:7:"contain";s:12:"minimal-logo";s:68:"http://wp.it-rays.net/octa/wp-content/uploads/2017/03/logo-light.png";s:12:"minimal-text";s:151:"Mauris mauris ante, blandit et, ultrices a, suscipit eget, quam. Integer ut neque. Vestibulum a velit eu ante scelerisque vulputate.Mauris mauris ante.";s:22:"address_minimal_footer";s:1:"1";s:20:"email_minimal_footer";s:1:"1";s:20:"phone_minimal_footer";s:1:"0";s:18:"fax_minimal_footer";s:0:"";s:14:"title-position";s:0:"";s:16:"page_head_height";s:0:"";s:10:"title_icon";s:0:"";s:19:"page_head_iconcolor";s:0:"";s:21:"page_head_iconbgcolor";s:0:"";s:15:"hide_page_title";s:0:"";s:10:"title_font";s:6:"Oswald";s:15:"title_font_size";s:0:"";s:18:"subtitle_font_size";s:0:"";s:17:"title_font_weight";s:0:"";s:11:"title-color";s:0:"";s:20:"page_title_fontcolor";s:0:"";s:19:"page_title_subcolor";s:0:"";s:20:"page_title_textcolor";s:0:"";s:18:"page_title_bgcolor";s:0:"";s:13:"page_title_bg";s:0:"";s:18:"page_title_bg_size";s:5:"cover";s:20:"page_title_bg_repeat";s:9:"no-repeat";s:22:"page_title_bg_position";s:6:"0 100%";s:19:"page_title_parallax";s:0:"";s:17:"page_head_overlay";s:0:"";s:16:"show_breadcrumbs";s:1:"1";s:17:"breadcrumbs_style";s:0:"";s:17:"breadcrumbs_align";s:10:"text-right";s:18:"breadcrumbs_prefix";s:0:"";s:18:"breadcrumbs_suffix";s:0:"";s:18:"mobile_breadcrumbs";s:1:"1";s:19:"breadcrumbs_bgcolor";s:0:"";s:17:"breadcrumbs_color";s:0:"";s:9:"body_font";s:9:"Open Sans";s:14:"body_font_size";s:0:"";s:16:"body_font_weight";s:0:"";s:16:"body_line_height";s:0:"";s:9:"menu_font";s:9:"Open Sans";s:14:"menu_font_size";s:0:"";s:16:"menu_font_weight";s:4:"bold";s:15:"menu_text_style";s:10:"capitalize";s:13:"headings_font";s:6:"Oswald";s:20:"headings_font_weight";s:3:"400";s:19:"headings_text_style";s:0:"";s:12:"custom_fonts";s:1:"0";s:18:"custom_font_name_0";s:0:"";s:17:"custom_font_ttf_0";s:0:"";s:17:"custom_font_eot_0";s:0:"";s:17:"custom_font_svg_0";s:0:"";s:18:"custom_font_woff_0";s:0:"";s:17:"custom_font_css_0";s:0:"";s:7:"favicon";s:0:"";s:9:"analytics";s:0:"";s:10:"tags_limit";s:2:"10";s:11:"wp_jpg_qlty";s:2:"80";s:8:"show_top";s:1:"1";s:15:"img_placeholder";s:1:"1";s:13:"page_comments";s:0:"";s:10:"404_layout";s:1:"1";s:8:"404_menu";s:27:"location-404 Not Found Menu";s:6:"404_bg";s:0:"";s:10:"404_poster";s:0:"";s:13:"404_video_mp4";s:0:"";s:14:"404_video_webm";s:0:"";s:13:"404_video_ogv";s:0:"";s:10:"custom_css";s:0:"";s:9:"custom_js";s:0:"";s:16:"show_sliding_bar";s:1:"1";s:13:"sbar_position";s:5:"right";s:24:"sliding_bar_content_push";s:1:"1";s:19:"sliding_bar_sidebar";s:6:"side-5";s:29:"number_of_sliding_bar_columns";s:2:"12";s:21:"sliding_bar_on_mobile";s:0:"";s:15:"sliding_padding";s:5:"80|80";s:16:"sliding_bg_color";s:7:"#222222";s:14:"sliding_bg_img";s:0:"";s:19:"sliding_bg_position";s:6:"0 100%";s:17:"sliding_bg_repeat";s:0:"";s:15:"sliding_bg_size";s:5:"cover";s:26:"sliding_bar_headings_color";s:7:"#ffffff";s:29:"sliding_bar_heading_font_size";s:0:"";s:22:"sliding_bar_font_color";s:7:"#cccccc";s:29:"sliding_bar_content_font_size";s:2:"13";s:12:"btn_position";s:9:"in_header";s:13:"sbar_btn_icon";s:19:"fa fa-align-justify";s:17:"sbar_btn_tgl_icon";s:11:"fa fa-times";s:18:"sbar_btn_icon_size";s:2:"14";s:14:"sbar_btn_shape";s:0:"";s:17:"sbar_btn_bg_color";s:4:"#222";s:14:"sbar_btn_color";s:7:"#ffffff";s:21:"sbar_btn_tgl_bg_color";s:4:"#222";s:18:"sbar_btn_tgl_color";s:4:"#fff";s:9:"blogstyle";s:8:"lg-image";s:12:"masonry_cols";s:1:"6";s:9:"grid_cols";s:1:"3";s:15:"blog_image_size";s:16:"blog-large-image";s:10:"it_excerpt";s:2:"50";s:12:"blog_sidebar";s:5:"right";s:18:"blog_sidebar_style";s:7:"default";s:10:"pager_type";s:1:"1";s:11:"pager_style";s:6:"style1";s:14:"pager_position";s:8:"centered";s:14:"load_more_text";s:0:"";s:19:"blog_single_sidebar";s:5:"right";s:25:"blog_single_sidebar_style";s:7:"default";s:18:"singlepostimg_size";s:5:"large";s:16:"singlepostimg_on";s:1:"1";s:15:"single_title_on";s:1:"1";s:12:"post_icon_on";s:1:"1";s:13:"singledate_on";s:1:"1";s:15:"singleauthor_on";s:1:"1";s:17:"singlecategory_on";s:1:"1";s:16:"singlecontent_on";s:1:"1";s:13:"singletags_on";s:1:"1";s:17:"singleprevnext_on";s:1:"1";s:18:"singleauthorbox_on";s:1:"1";s:16:"singlecomment_on";s:1:"1";s:16:"singlerela

wp-content/plugins/octa-portfolio/inc/config.php
<?php

class OCTA_Config {
    
    public $settings;
    
    public function __construct() {
        add_action('init', array(&$this, 'configArray'));
    }
    
    public function configArray() {
        $this->settings = array(
            array(
                "name" => "id",
                "title" => "id",
                "data_type" => "mediumint(9)",
                "type" => "int",
                "auto" => "AUTO_INCREMENT",
                "not_null" => "NOT NULL",
            ),
            /* Start Naming */
            array(
                "name" => "title",
                "title" => "Title",
                "data_type" => "text",
                "type" => "text",
                "section" => "oc_naming",
                "not_null" => " NULL",
                "description" => "Insert the grid title.",
            ),
            array(
                "name" => "alias",
                "title" => "Alias",
                "data_type" => "text",
                "type" => "text",
                "class" => "alias_txt",
                "section" => "oc_naming",
                "not_null" => " NULL",
                "description" => "Insert the grid alias.",
            ),
            array(
                "name" => "shortcode",
                "title" => "Shortcode",
                "data_type" => "text",
                "type" => "disabledtext",
                "class" => "shortcode_txt",
                "section" => "oc_naming",
                "not_null" => " NULL",
                "description" => "Shortcode will be generated automatically once you type the alias.",
            ),
            array(
                "name" => "extra_class",
                "title" => "Extra CSS Class",
                "data_type" => "text",
                "type" => "text",
                "section" => "oc_naming",
                "not_null" => " NULL",
                "description" => "Add extra css class name and refer to it in custom CSS.",
            ),
            array(
                "name" => "oldalias",
                "data_type" => "text",
                "type" => "hidden",
                "section" => "oc_naming",
                "not_null" => " NULL",
            ),
            /* Start Source */
            array(
                "name" => "post_type",
                "title" => "Post Types",
                "data_type" => "text",
                "type" => "multidropdown",
                "choices" => oc_post_types(),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Select from available post types.",
                "std" => "octa",
            ),
            array(
                "name" => "select_taxonomy",
                "title" => "Post Categories",
                "data_type" => "text",
                "type" => "taxsdropdown",
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Select from available post types.",
            ),
            /*array(
                "name" => "cat_rel",
                "title" => "Category Relation",
                "data_type" => "text",
                "type" => "radio",
                "choices" => array(
                    'OR' => 'OR',
                    'AND' => 'AND',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                'std' => 'OR',
            ),*/
            array(
                "name" => "maximum_entries",
                "title" => "Maximun Entries",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Leave it (-1) if you want to show all portfolio posts.",
                "placeholder" => "-1",
                "std" => "-1",
                "min" => "-1",
                "max" => "100",
            ),
            array(
                "name" => "order_by",
                "title" => "Order By ",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'date' => 'Date',
                    'title' => 'Title',
                    'id' => 'Id',
                    'name' => 'Name',
                    'modified' => 'Modified',
                    'author' => 'Author',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Select the grids order from the available options.",
            ),
            array(
                "name" => "order_type",
                "title" => "Order Type",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'desc' => 'Descending',
                    'asc' => 'Ascending',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Order posts ascending or descending.",
            ),
            array(
                "name" => "image_source",
                "title" => "Image Source Type",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'full' => 'Full',
                    'thumbnail' => 'Thumbnail',
                    'medium' => 'Medium',
                    'medium_large' => 'Medium Large',
                    'large' => 'Large',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Choose what image size to be set to the image in the grid items.",
            ),
            array(
                "name" => "enable_pagination",
                "title" => "Enable pagination?",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Enable or disable pagination.",
                "std" => "",
            ),
            array(
                "name" => "items_per_page",
                "title" => "Posts Per Page",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Number of posts for each page.",
                "placeholder" => "-1",
                "std" => "-1",
                "min" => "-1",
                "max" => "100",
                'dependency' => array(
                    'element' => 'enable_pagination',
                    'value' => '1'
                ),
            ),
            array(
                "name" => "pagination_type",
                "title" => "pagination type",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'numeric' => 'Numeric',
                    'nextprev' => 'Older - Newer',
                    'loadmore' => 'Load More',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Choose pagination type from list above.",
                'dependency' => array(
                    'element' => 'enable_pagination',
                    'value' => '1'
                ),
            ),
            array(
                "name" => "pagination_alignment",
                "title" => "pagination Alignment",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'center' => 'Center',
                    'left' => 'Left',
                    'right' => 'Right',
                ),
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Align the pager below the grids horizontally.",
                'dependency' => array(
                    'element' => 'enable_pagination',
                    'value' => '1'
                ),
            ),
            array(
                "name" => "load_more_button_text",
                "title" => "Load More Button Text",
                "data_type" => "text",
                "type" => "text",
                "section" => "oc_source",
                "not_null" => " NULL",
                "description" => "Align the pager below the grids horizontally.",
                'dependency' => array(
                    'element' => 'pagination_type',
                    'value' => 'loadmore'
                ),
                "placeholder" => "Load More",
                "std" => "Load More",
            ),
            /* Start Grid Setting */
            array(
                "name" => "grid_layout",
                "title" => "Grid Layout",
                "data_type" => "text",
                "type" => "dropdown",
                "class" => "select_boxes",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "choices" => array(
                    'grid' => 'Even',
                    'masonry' => 'Masonry',
                    'justify' => 'Dynamic Width',
                    'onecolumn' => 'One Column',
                    'slider' => 'Slider',
                ),
                "description" => "Choose how the grid will appear for usrs.",
                "std" => "grid",
            ),
            array(
                "name" => "img_ratio",
                "title" => "Media Ratio X:Y",
                "data_type" => "text",
                "type" => "twonumber",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "placeholder" => "5|4",
                "std" => "5|4",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'grid'
                ),
            ),
            array(
                "name" => "rtl",
                "title" => "RTL",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "0",
            ),
            array(
                "name" => "slider_type",
                "title" => "Slider Type",
                "data_type" => "text",
                "type" => "dropdown",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "choices" => array(
                    'horizontal' => 'Horizontal',
                    'vertical' => 'Vertical',
                ),
                "std" => "horizontal",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "slide_to_show",
                "title" => "Slides To Show",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "placeholder" => "4",
                "description" => "Enter the number of slides to be shown.",
                "std" => "4",
                "min" => "1",
                "max" => "10",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "slide_to_scroll",
                "title" => "Slide To Scroll",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "placeholder" => "1",
                "description" => "Number of slides that will be scrolled.",
                "std" => "1",
                "min" => "1",
                "max" => "10",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "slide_speed",
                "title" => "Slide Speed",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "placeholder" => "500",
                "description" => "Speed of the slider in milliseconds.",
                "std" => "500",
                "min" => "0",
                "max" => "1000",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "fade",
                "title" => "Fade ?",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "auto_play",
                "title" => "Auto Play ? ",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "show_arrows",
                "title" => "Show Arrows ? ",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "1",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "show_bullets",
                "title" => "Show Bullets ? ",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "infinite",
                "title" => "Infinite ? ",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "std" => "1",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => 'slider'
                ),
            ),
            array(
                "name" => "preloader",
                "title" => "Pre-loader",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    '-1' => 'None',
                    'cp-round' => 'Spinner Round',
                    'cp-pinwheel' => 'Spinner Pinwheel',
                    'cp-balls' => 'Spinner Balls',
                    'cp-bubble' => 'Spinner Bubble',
                    'cp-flip' => 'Spinner Flip',
                    'cp-hue' => 'Spinner Hue',
                    'cp-skeleton' => 'Spinner Skeleton',
                    'cp-eclipse' => 'Spinner Eclipse',
                    'cp-boxes' => 'Spinner Boxes',
                    'cp-morph' => 'Spinner Morph',
                ),
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "description" => "Select a preloading styles that appears before all grid items completely loaded.",
            ),
            array(
                "name" => "number_of_columns",
                "title" => "Columns Per Row",
                "data_type" => "text",
                "type" => "dropdown",
                "section" => "oc_gnrlsetting",
                "choices" => array(
                    '2' => '2 Columns',
                    '3' => '3 Columns',
                    '4' => '4 Columns',
                    '5' => '5 Columns',
                    '6' => '6 Columns',
                ),
                "std" => "4",
                "not_null" => " NULL",
                "description" => "Items per row that will be shown horizontally.",
                'dependency' => array(
                    'element' => 'grid_layout',
                    'value' => array('grid','masonry','slider')
                ),
            ),
            array(
                "name" => "item_spacing",
                "title" => "Item Spacing",
                "data_type" => "int",
                "type" => "number",
                "section" => "oc_gnrlsetting",
                "not_null" => " NULL",
                "description" => "Enter the Spacing between grid items.",
                "placeholder" => "15",
                "std" => "15",
                "min" => "0",
                "max" => "100",
            ),
            /* Start Skins & Styles */
            array(
                "name" => "choose_skin",
                "title" => "Choose Skin",
                "data_type" => "text",
                "type" => "radio",
                "choices" => array(
                    'kara' => 'Kara',
                    'paleo' => 'Paleo',
                    'hub' => 'Hub',
                    'ivy' => 'Ivy',
                    'sublime' => 'Sublime',
                    'krosh' => 'Krosh',
                    'impress' => 'Impress',
                    'transit' => 'Transit',
                    'resort' => 'Resort',
                    'gemini' => 'Gemini',
                    'neuron' => 'Neuron',
                    'onair' => 'OnAir',
                    'solo' => 'Solo',
                    'rotato' => 'Rotato',
                    'focus' => 'Focus',
                    'agent' => 'Agent',
                    'rolly' => 'Rolly',
                    'mass' => 'Mass',
                    'marbele' => 'Marble',
                    'astro' => 'Asrto',
                    'zilla' => 'Zilla',
                ),
                "class" => "portfolio-item",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "p-style1",
            ),
            array(
                "name" => "show_title",
                "title" => "Show Title",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "1",
            ),
            array(
                "name" => "show_categories",
                "title" => "Show Taxonomies",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "1",
            ),
            array(
                "name" => "show_link_to_post",
                "title" => "Show Link to post",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "1",
            ),
            array(
                "name" => "show_zoom_image",
                "title" => "Show Zoom Image",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "1",
            ),
            array(
                "name" => "show_excerpt",
                "title" => "Show Excerpt",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_skins",
                "not_null" => " NULL",
                "std" => "",
            ),
            /* Start Nav Filter */
            array(
                "name" => "nav_filter",
                "title" => "Nav-Filter",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'style1'    => 'Round Boxes',
                    'style2'    => 'Bottom Bordered',
                    'style3'    => 'Boxed Bottom Bordered',
                    'style4'    => 'Simple Nav',
                    'style5'    => 'Top Bottom Borders',
                    'none'      => 'None',
                ),
                "section" => "oc_nav",
                "not_null" => " NULL",
                "description" => "Select Navigation filter style that appears at the top of the grids.",
            ),
            array(
                "name" => "show_all",
                "title" => "Show All Button",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_nav",
                "not_null" => " NULL",
                "description" => "Show / Hide (All) button.",
                "std" => "1",
            ),
            array(
                "name" => "all_text",
                "title" => "Show All Button Text",
                "data_type" => "text",
                "type" => "text",
                "section" => "oc_nav",
                "not_null" => " NULL",
                "std" => "All",
                'dependency' => array(
                    'element' => 'show_all',
                    'value' => '1'
                ),
            ), 
            array(
                "name" => "nav_layout",
                "title" => "Layout Option",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'inline' => 'List',
                    'dropdown' => 'DropDown',
                ),
                "section" => "oc_nav",
                "not_null" => " NULL",
            ),
            array(
                "name" => "els_num",
                "title" => "Show Number of Elements",
                "data_type" => "text",
                "type" => "checkbox",
                "section" => "oc_nav",
                "not_null" => " NULL",
                "std" => "0",
            ),
            array(
                "name" => "num_style",
                "title" => "Numbers Style",
                "data_type" => "text",
                "type" => "dropdown",
                "choices" => array(
                    'inline' => 'Inline',
                    'popup' => 'Popup',
                ),
                "section" => "oc_nav",
                "not_null" => " NULL",
            ),
            /* Start Custom Css */
            /*array(
                "name" => "custom_css",
                "title" => "Custom CSS",
                "data_type" => "text",
                "type" => "textarea",
                "class" => "custom_css",
                "section" => "oc_css",
                "not_null" => " NULL",
                "description" => "Add custom css code for this grid to give it a custom style.",
            ), */
        );
        return $this->settings;
    }
    
}
new OCTA_Config();
